ls1intum/Artemis
# [Exam] Enhancement/exam/exercise layout

### Checklist

### Motivation and Context


Separates the problem statement from the students input. 
Closes #1589 #1588 #1587 

### Description


- Use `resizable-container.component` to separate the problem statement and make it resizable/collapsable for Text, Modeling and File Upload Exercises
- Use `media (max-width: 991.98px)` for smaller screens
- Removed border and header for exercise submissions
- Small changes for team exercises and modeling exercise

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Create Text, Modeling, FileUpload Exercises and set start date to now and due date in a few minutes
4. Participate in Exercises and check if the Problem Statement is resizable/collapsable
4.1 Check if for smaller screens the Problem Statement aligns above the students input and is not resizable/collapsable
5. After Due Date see your Submission and check if Problem Statement is resizable/collapsable
5.1 Check if for smaller screens the Problem Statement aligns above the students input and is not resizable/collapsable
6. Check also for team exercises

Note: `resizable-container.component` is also used in `text-submission-assessment` 
7. Check if assessment instructions align above students submission for smaller screens

### Screenshots


File Upload:
<img width="1277" alt="image" src="https://user-images.githubusercontent.com/41108487/85124216-d4b11600-b229-11ea-963f-7008623cfb16.png">


Modeling:
<img width="1271" alt="image" src="https://user-images.githubusercontent.com/41108487/85124021-869c1280-b229-11ea-80af-e52225d2b192.png">



Text:
<img width="1268" alt="image" src="https://user-images.githubusercontent.com/41108487/85124153-b814de00-b229-11ea-8b6f-bbf5191cf5c1.png">


Smaller devices:
<img width="766" alt="image" src="https://user-images.githubusercontent.com/41108487/85124109-a7fcfe80-b229-11ea-8104-b993aa244992.png">

<img width="779" alt="image" src="https://user-images.githubusercontent.com/41108487/85124178-c3680980-b229-11ea-99f6-4fe9603b853f.png">

<img width="770" alt="image" src="https://user-images.githubusercontent.com/41108487/85124252-e2669b80-b229-11ea-8e02-4d74bb29d72d.png">




Team exercises:
<img width="1239" alt="image" src="https://user-images.githubusercontent.com/41108487/85123912-5e141880-b229-11ea-905f-18f79e69f9c9.png">

<img width="1245" alt="image" src="https://user-images.githubusercontent.com/41108487/85123937-69674400-b229-11ea-8d0e-047ce4cc541a.png">

Team exercises on smaller devices:
<img width="759" alt="image" src="https://user-images.githubusercontent.com/41108487/85124307-fa3e1f80-b229-11ea-9dca-a416dfbdf32d.png">

<img width="747" alt="image" src="https://user-images.githubusercontent.com/41108487/85124349-104be000-b22a-11ea-8759-6a85bc3974b4.png">
